CardForm/src/test/java/com/braintreepayments/cardform/view/CardEditTextTest.java

```diff
@@ -81,6 +84,68 @@ public void testUnknown() {
         helper("1", "111 1111 1111 1111111", R.drawable.bt_ic_unknown, 4, 8, 12);
     }
 
+    @Test
+    public void showsMaskedInputWhenValidAndFocusChanges() {
+        mView.setMasked(true);
+        type("4111111111111111");
+        mView.onFocusChanged(false, 1, null);
+
+        assertNotNull(mView.getTransformationMethod());
+        assertTrue(mView.getTransformationMethod() instanceof CardNumberTransformation);
```

Comment:
Is the transformation only in the UI or can you assert on the value of the EditText?

Reply:
It's only in the UI, `getText` will return the unmasked value. We also have tests for the transformation so this seemed like a good middle ground.